### ISSUE TITLE
install: missing file operand

### ISSUE THREAD

**Issue by jounathaen** [ParaStation/psmgmt#2]
I tried to build psmgmt 5.1.26-0 with the following command:
```
./configure --disable-psconfig --disable-pspmix --prefix /tmp/asdf
make
make install
```
And then I end up with the following error (excerpt):
```
make[3]: Leaving directory `/tmp/psmgmt/psmgmt-5.1.26-0/plugins/psmom/pbs'
Making install in psmom
make[3]: Entering directory `/tmp/psmgmt/psmgmt-5.1.26-0/plugins/psmom/psmom'
make[4]: Entering directory `/tmp/psmgmt/psmgmt-5.1.26-0/plugins/psmom/psmom'
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/jobs
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/nodefiles
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/temp
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/undelivered
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/account
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/backup
/usr/bin/install -c -d /tmp/asdf/var/spool/parastation/scripts
/usr/bin/install -c -d
/usr/bin/install: missing file operand
Try '/usr/bin/install --help' for more information.
make[4]: *** [install-exec-local] Error 1
make[4]: Leaving directory `/tmp/psmgmt/psmgmt-5.1.26-0/plugins/psmom/psmom'
make[3]: *** [install-am] Error 2
make[3]: Leaving directory `/tmp/psmgmt/psmgmt-5.1.26-0/plugins/psmom/psmom'
make[2]: *** [install-recursive] Error 1
```
Seems like there is a erronous whitespace in some autotools file?

**Comment by jounathaen** [comment]
seems to result from a missing `pamdir`...

**Comment by moschny** [comment]
Here's a suggestion for a patch:

```diff
--- a/plugins/psmom/psmom/Makefile.am
+++ b/plugins/psmom/psmom/Makefile.am
@@ -55,8 +55,12 @@ install-exec-local:
        $(INSTALL) -d $(DESTDIR)$(localstatedir)/spool/parastation/account
        $(INSTALL) -d $(DESTDIR)$(localstatedir)/spool/parastation/backup
        $(INSTALL) -d $(DESTDIR)$(localstatedir)/spool/parastation/scripts
+if PAM_ENABLED
        $(INSTALL) -d $(DESTDIR)$(pamdir)
        $(LN_S) pam_pspam.so $(DESTDIR)$(pamdir)/pam_psmom.so
+endif
 
 uninstall-local:
+if PAM_ENABLED
        $(RM) $(DESTDIR)$(pamdir)/pam_psmom.so
+endif
```
auto* files need to be regenerated after applying the patch with `autoreconf -vif`.

**Comment by moschny** [comment]
Fixed by @neicker  in a similar way in dfe559c.
